### PR TITLE
[SPARK-5275] [Streaming] include python source code

### PR DESCRIPTION
**Pull request by davies**
Include the python source code into assembly jar.

cc @mengxr @pwendell

### CODE REVIEW

File: streaming/pom.xml [hunk 1]
```diff
@@ -106,5 +106,13 @@
         </executions>
       </plugin>
     </plugins>
+    <resources>
+      <resource>
+      <directory>../python</directory>
```
Comment (pwendell) [inline streaming/pom.xml]: Should this be indented another level?

**Comment by pwendell** [comment]
LGTM with just one minor style comment. @davies I'm assuming you tested this?

**Comment by davies** [comment]
@pwendell I'm working on double-check it.

What's the purpose of include the python file into assembly jar? Those are already included in bin-package, also we didn't include the python files in assembly jar generated by `sbt/sbt assembly`.

**Comment by davies** [comment]
@pwendell I had confirmed that these files will be included in ./dist/lib/spark-assembly-XXX-SNAPSHOT-hadoop.jar

**Comment by pwendell** [comment]
Thanks Davies, I'm merging this.

**Pull request closed without merging**